src/prefixes.rs

```diff
@@ -93,6 +93,27 @@ pub async fn prefix_remove(
     Ok(())
 }
 
+/// Delete all of your user-specific prefixes
+#[poise::command(rename = "reset", prefix_command, slash_command)]
+pub async fn prefix_reset(ctx: Context<'_>) -> Result<(), Error> {
+    let user_id = ctx.author().id.0 as i64;
+    let prefixes_deleted = sqlx::query!("DELETE FROM prefix WHERE user_id = ?", user_id)
+        .execute(&ctx.data().database)
+        .await?
+        .rows_affected();
+
+    let msg = if prefixes_deleted == 0 {
+        "You have no prefixes to deleted".to_string()
+    } else if prefixes_deleted == 1 {
+        "1 prefix was deleted".to_string()
+    } else {
+        format!("{} prefixes were deleted", prefixes_deleted)
+    };
```

Comment:
I know that's ugly. Should I, as an alternative, just not display the number of prefixes removed? Or use the classical `prefix(es)`? Or what's the way you'd suggest?

Reply:
Nah it's fine. It's less ugly than the classical `prefix(es)` spelling in my opinion :p

Btw it should be "You have no prefixes to **delete**"

Reply:
I'll fix that typo and run cargo-sqlx prepare. give me a minute